app/avo/resources/oidc_provider_resource.rb

```diff
@@ -0,0 +1,23 @@
+class OIDCProviderResource < Avo::BaseResource
+  self.title = :issuer
+  self.includes = []
+  self.model_class = ::OIDC::Provider
+  # self.search_query = -> do
+  #   scope.ransack(id_eq: params[:q], m: "or").result(distinct: false)
+  # end
+
+  action RefreshOIDCProvider
+
+  # Fields generated from the model
+  field :issuer, as: :text, link_to_resource: true
+  field :configuration, as: :nested do
+    visible_on = %i[edit new]
+    OIDC::Provider::Configuration.then { (_1.required_attributes + _1.optional_attributes) - fields.map(&:id) }.each do |k|
```

Comment:
Agreed it's not blocking. Just raising that this can be iterated on so other admins are able to create/modify providers more easily when it comes to that point.

Reply:
The modification is done automatically via an action that fetches from the well-known URLs